### PR TITLE
Support573

### PR DESCRIPTION
**Pull request by samgnaniah**
Adds Azure Service Bus docs into the doc site.

### CODE REVIEW

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+This section describes how to configure WSO2 Micro Integrator to connect with Azure Service Bus.
+
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
+
+## Setting up the Micro Integrator with Azure Service Bus
+
+Follow the instructions below to set up and configure.
+
+1.  Download [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
```
Comment (indikasampath2000) [inline en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:11]: The Azure Service Bus is a messaging service available in the Azure Cloud. There is nothing like to download. But you have to configure it as I have explained in the document.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Connecting to Azure Service Bus
+
+This section describes how to configure WSO2 Micro Integrator to connect with Azure Service Bus.
+
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
```
Comment (indikasampath2000) [inline en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:5]: We should say something like this.
The Azure Service Bus complies with Both AMQP 1.0 and JMS 2.0 [1]. The Micro Integrator uses its inbuilt JMS transport to send and receive messages from the Azure Service Bus. The configurations are similar to connecting with other JMS brokers.

[1] https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview#compliance-with-standards-and-protocols

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        parameter.provider_url = "conf/jndi.properties"
+        parameter.connection_factory_name = "SBCF"
+        parameter.connection_factory_type = "queue"
+        ```
+3.  Remove any existing Apache ActiveMQ client JAR files from the `MI_HOME/dropins/` and `MI_HOME/lib/` directories.  
+4.  Remove the below line from the `MI_HOME/conf/etc/launch.ini` file.  
+
+    ```text
+    javax.jms,\
+    ```
```
Comment (indikasampath2000) [inline en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:52]: I am not sure whether these steps are required.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+5.  Start Azure Service Bus. For instructions, see the [Azure Service Bus Documentation](https://docs.microsoft.com/en-us/azure/service-bus-messaging/).
```
Comment (indikasampath2000) [inline en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:53]: There is nothing to start in Azure Service Bus after config it properly. We have to mention how to configure similar to the instructions given in the documentation.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+The integration between this message broker and WSO2 Micro Integrator is straightforward. The guaranteed delivery integration pattern must be used to implement the mediation flow.
+
+## Setting up the Micro Integrator with Azure Service Bus
+
+Follow the instructions below to set up and configure.
+
+1.  Download [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/).
+2.  Download and install WSO2 Micro Integrator.
+
+### Setting up Azure Service Bus
```
Comment (indikasampath2000) [inline en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:14]: Azure Service Bus configuration must go here.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md (hunk 1 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md]: Fixed.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md (hunk 2 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md]: Added this in.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md (hunk 3 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md]: Checked and removed.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md (hunk 4 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md]: Removed this.

File: en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md (hunk 5 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:14]: Added new content here.

Comment (indikasampath2000) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:14]: We have to mentioned how to configure a queue in Azure Service Bus to work with the following synapse config. Can you add a small paragraph and give this link? https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal

Comment (samgnaniah) [reply en/micro-integrator/docs/setup/brokers/configure-with-azureservicebus.md:14]: Fixed.

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**